operator/test/test.sh

```diff
@@ -149,17 +149,27 @@ test_pipelines() {
   # change ubuntu image to ubi to avoid dockerhub registry pull limit
   PIPELINE_RUN=$(curl --fail --silent "$BASE_URL/examples/v1beta1/$manifest" | sed 's|ubuntu|registry.access.redhat.com/ubi8/ubi-minimal:latest|' | sed '/serviceAccountName/d' | KUBECONFIG="$KUBECONFIG_KCP" kubectl create -f -)
   echo "$PIPELINE_RUN"
-  
-  KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
+
+  # KUBECONFIG="$KUBECONFIG_KCP" kubectl wait --for=condition=Succeeded  PipelineRun --all --timeout=60s >/dev/null
```

Comment:
Delete that line.

Reply:
I fixed the DiskPressure issue of hypershift's hosted cluster. I removed debuging lines in the `test.sh`. This line is uncommend.